alternator/executor.cc

```diff
@@ -84,6 +84,21 @@ std::string make_jsonable::to_json() const {
     return rjson::print(_value);
 }
 
+json::json_return_type make_streamed(rjson::value&& value) {
+    // CMH. json::json_return_type uses std::function, not noncopyable_function. 
+    // Need to make a copyable version of value. Gah.
+    auto rs = make_shared<rjson::value>(std::move(value));
+    std::function<future<>(output_stream<char>&&)> func = [rs](output_stream<char>&& os) mutable -> future<> {
+        auto los = std::move(os);
+        auto lrs = std::move(rs);
+        co_await rjson::print(*lrs, los);
```

Comment:
If print() fails, we never call close(). That might not be safe?
My implementation of this function had
```
return print(os, v).finally([&os, v] { return os.close(); });
```
(I didn't use corotines for this simple case).

Reply:
If we get an exception here (oom?) we're in bad trouble, because we have already written all headers and potentially part of content. We cannot generate a proper response in any way (as in HTTP error). I don't think it matters much, but  should both add logging and cleanup, yes.